experiment/slack-oncall-updater/main.go

```diff
@@ -127,21 +122,42 @@ func main() {
 		log.Fatalf("Failed to get Slack token: %v", err)
 	}
 
-	oncall, err := getCurrentOncaller()
+	oncallers, err := getCurrentOncallers()
 	if err != nil {
-		log.Fatalf("Failed to find current oncaller: %v", err)
+		log.Fatalf("Failed to find current oncallers: %v", err)
 	}
-	log.Printf("Current oncaller: %s\n", oncall)
-
-	userID, err := mapGitHubToSlack(oncall)
-	if err != nil {
-		log.Fatalf("Failed to get Slack ID: %v", err)
+	log.Printf("Current oncallers: %s\n", oncallers)
+
+	failed := false
+	for rotation, user := range oncallers {
+		// skip rotations that have not yet added their slack ID
+		// this tool is not required
+		groupID, exists := rotationToSlackGroup[rotation]
```

Comment:
FYI currently the json key doesn't map to the rotation.
i.e. `test-infra` is named `testinfra` in the json blob.

we will also need to change that export alias.

Reply:
we don't, I just have the wrong key in the map above then.

Reply:
that works as well

Reply:
should be fixed